cpp/src/quantiles/quantile.cu

```diff
@@ -62,19 +64,29 @@ struct quantile_functor {
       return output;
     }
 
-    auto d_input  = column_device_view::create(input);
+    auto d_input  = column_device_view::create(input, stream);
     auto d_output = mutable_column_device_view::create(output->mutable_view());
```

Comment:
Don't we need to provide stream here?

Reply:
Yes. If the column has children (like dictionary does), then you need to pass a stream. It is harmless to pass otherwise.